src/drivers/driver_options.go

```diff
@@ -0,0 +1,8 @@
+package drivers
+
+// DriverOptions is the options passed to GetDriver
+type DriverOptions struct {
+	DriverOverride         string
+	OriginHostnameOverride string
```

Comment:
Not sure we need the suffix `Override` on these options.

Reply:
I think `Override` is important as it these are overriding what we detect from `OriginURL`. Otherwise the order of importance of the options I think would be harder to understand. Open to using other things here, but that is what I am trying to make clear

Reply:
My thinking was along the lines of: if a driver name is given here, use it. Otherwise Git Town tries to determine it by itself.